fedora_messaging/cli.py

```diff
@@ -150,3 +167,33 @@ def consume(exchange, queue_name, routing_key, callback, app_name):
                 str(e.reason),
             )
             sys.exit(e.exit_code)
+
+
+@cli.command()
+@click.option("--publish-exchange", help=_publish_exchange_help)
+@click.option("--body-key", help=_body_key_help)
+@click.option("--body-value", help=_body_value_help)
```

Comment:
I wonder if it would be nicer to accept the path to a file that contains JSON. If we did that, we'd need to come up with a format for a JSON-serialized message (message topic, headers, body, etc). That seems generally useful anyway for users who want to create message test fixtures.

What do you think, @abompard?

Reply:
Yeah I agree, that'd be useful anyway.